.travis.yml

```diff
@@ -63,6 +63,7 @@ matrix:
         - pip install -U 'future==0.16.0'
         - pip install -U 'Jinja2>=2.10.1,<2.11'
         - pip install -U 'intelhex>=1.3,<=2.2.1'
+        - pip install -U 'mbed-host-tests>=1.8.0,<2.0.0'
```

Comment:
Isn't "mbed-host-tests" part of the legacy toolset? Do we need to depend on them to copy a file to a DAPLINK device? What if we were to rewrite the new tools to make them standalone: we couldn't depend on this library.

This looks like the first python dependency we are adding to mbedtools at run-time, which we should think very carefully about.

Reply:
`mbed-tools` doesn't depend on `mbed-host-tests` anymore.